### ISSUE TITLE
Status reports of development progress and prototype

### ISSUE THREAD

**Issue by desputinski** [tcfev/forDem#165]
Hi all, I created this thread to make it more transparent for everybody where we stand on the development progress in total. Which Parts of the development are already finished, and which are not.
Does it make sense to have an issue like this?

**Comment by armantorkzaban** [comment]
Go on, of course it makes sense. We can then link it in the README

**Comment by desputinski** [comment]
Hm, not sure about how the status is supposed to be presented, in form of a forum like in this thread? But to fill it, I think the programmers themselves have to write something once in a while... (also interesting for everybody not involved in programming would be who the programmers)

**Comment by armantorkzaban** [comment]
I don't know if this is the right place for this announcement or not but after a series of negotiations, we are making progress in joining forces with a group of north American Iranian entrepreneurs who want to make large-scale WiFi-Direct Mesh network a reality and aligned with our e-democracy goals.

**Comment by lukas-h** [comment]
Hi,
if/once we start with proper iterations/sprints in an agile development cycle, we should make sprint plannings (priorisation of features), sprint refinements (in the middle of the sprint), and **sprint demos**, where we will state our progress to everyone.

Wdyt?

**Comment by desputinski** [comment]
Good idea.  How far is programming of the basic app itself yet? I mean the GUI with basic functionalities?

**Comment by lukas-h** [comment]
Hard to explain. We have a codebase from another open source, that we can borrow. It offers some of the functionality.
But the key functionalities (polling, discussions, etc.) are not implemented yet.

**Comment by desputinski** [comment]
Ok, thanks. What about the status on the other key functionalities of mobilization, so sharing of dates for political action, integrating a map for the event etc?

**Comment by desputinski** [comment]
Hm, I just downloaded the "code" of forDem and there is nothing inside. Is it the case, that the actual programming is taking place elsewhere, or is it just, that I don't understand github?

**Comment by armantorkzaban** [comment]
Much of the work is taking place here at the moment (and also locally): 
https://github.com/tcfev/mastodon-csharp
https://github.com/tcfev/fordem-backend/
